### DISCUSSION TITLE
The example "models_first_person_maze.c" should work as expected.

### DISCUSSION THREAD

**Discussion by wlxmhls** [raysan5/raylib#2653] [Q&A] [answered]
Hello, I am trying the example code "models_first_person_maze.c". It uses the first person camera. 
When I move the camera using 'A/W/S/D' key, the camera seems shaking up and down. That looks the camera is in the sea with heavy waves.
It should not happen in the solid ground.

**Comment by raysan5** [comment, marked as answer]
@wlxmhls You can modify those parameters in the `rcamera` module and recompile raylib as required or implement your custom camera system. The provided example is just an example, not expected to be used in real production.

**Reply by wlxmhls** [reply]
I am a newbie in 3D programming. Probably it is beyond my ability.
